--- origsrc/gstreamer-1.2.6/lib/gst.rb	2013-08-05 13:36:41.414818100 -0500
+++ src/gstreamer-1.2.6/lib/gst.rb	2013-08-05 13:36:56.303669700 -0500
@@ -4,7 +4,6 @@ require 'glib2'
 base_dir = Pathname.new(__FILE__).dirname.dirname.expand_path
 vendor_dir = base_dir + "vendor" + "local"
 vendor_bin_dir = vendor_dir + "bin"
-GLib.prepend_environment_path(vendor_bin_dir)
 if vendor_bin_dir.exist?
   require 'pango'
   require 'gdk_pixbuf2'
